app/config/sources.inc.php

```diff
@@ -730,6 +730,10 @@
         'deadline'          => '2018-02-18',
         'supported_locales' => ['de', 'es', 'fr', 'pl', 'pt-BR'],
     ],
+    'emails/2018/firefox_learning_journey.lang' => [
+        'deadline'          => '2018-06-03',
```

Comment:
Actually, this doesn't match the date in the bug (Jun 1st). Do they really want a deadline on Friday? Monday would be definitely better…

Reply:
Yes, I purposely add the weekend in the schedule. The "production" of the localized copy starts on Monday. I will comment in the bug.